src/SourceBuild/Arcade/eng/common/templates/job/source-build-run-tarball-build.yml

```diff
@@ -160,6 +167,7 @@ jobs:
         additionalBuildArgs: --with-sdk /tarball/.dotnet
         buildArch: $(_BuildArch)
         container: $(_Container)
+        enablePoison: $(_EnablePoison)
```

Comment:
What value do we get from running in both scenarios (normal and bootstrapped)?  Is it sufficient to run in just one configuration?

Reply:
Yes, I think you're correct.  Bootstrapping is a little harder to reason about so I got rid of that one.